keras-dreambooth-sprint/README.md

```diff
@@ -1,95 +1,135 @@
 
-You can fine-tune on any concept that you want. Couple of inspirations for you: 
+Couple of inspirations for you: 
 
+TODO: it might be confusing to show examples not using keras. maybe worth for us to build some of the examples and share them
 1. Lowpoly World: This [model](https://huggingface.co/MirageML/lowpoly-world) generates lowpoly worlds 🤯🌍
 2. Future Diffusion: This [model](https://huggingface.co/nitrosocke/Future-Diffusion) generates images in futuristic sci-fi concepts 🤖
 3. Fantasy sword: This [model](https://huggingface.co/MirageML/fantasy-sword) generates swords for fantasy themed games 🧙‍♂️
 
-Are you looking to stand out from the crowd? If you partner up with an artist to fine-tune on a **********consentful********** concept, you’ll be eligible to submit your model under the `consentful` theme!
-
-If you need more pointers on Dreambooth implementation with Keras, you can check out [this repository](https://github.com/sayakpaul/dreambooth-keras). 
+## Compute
 
 [ADD DETAILS ABOUT COMPUTE SPONSORSHIP]
 
-### Dreambooth Diffusers Integration with KerasCV
+## Model Training
 
-As of now, inference and deployment options of `KerasCV` is limited, which `diffusers` library could come for a rescue. With only few lines of code, we can convert a `KerasCV` model into a `diffusers` one and use `diffusers`’ pipelines to perform inference. You can get more information [here](https://huggingface.co/docs/diffusers/main/en/using-diffusers/kerascv). Also check out [this Space](https://huggingface.co/spaces/sayakpaul/convert-kerascv-sd-diffusers) for converting your `KerasCV` model to `diffusers`one.
+You can find the notebook here and adjust it according to your own dataset 👇  
 
-```
-from diffusers import StableDiffusionPipeline
+TODO: The notebook below is a Google Drive. We should add it to the GitHub repo
 
-# checkpoint of the converted Stable Diffusion from KerasCV
-model_ckpt = "sayakpaul/text-unet-dogs-kerascv_sd_diffusers_pipeline"
-pipeline = StableDiffusionPipeline.from_pretrained(model_ckpt)
-pipeline.to("cuda")
+[Link to notebook](https://colab.research.google.com/drive/1Bh49YBYgaNc3X5aRzaMZNska4ErvU3lJ?usp=sharing). If you need more pointers on Dreambooth implementation with Keras, you can check out [this repository](https://github.com/sayakpaul/dreambooth-keras).
 
-unique_id = "sks"
-class_label = "dog"
-prompt = f"A photo of {unique_id} {class_label} in a bucket"
-image = pipeline(prompt, num_inference_steps=50).images[0]
-```
+Note/q: having a colab and then a repo could maybe be confusing?
+Note: maybe let's change the repo_id in push_to_hub_keras to push to the organization.
```

Comment:
we left it up to people, that's why we have tags in readmes.

Reply:
Ah, that was a bit confusing to me

> We will be hosting our demos in this organization on Hugging Face Hub: [keras-dreambooth](https://huggingface.co/keras-dreambooth), send a request to join [here](https://huggingface.co/organizations/keras-dreambooth/share/RMocthadPgpxxUDHtAesrbBzieDLgUfPmv) if you’d like to make a submission 🙂

So IIUC, users can upload the models to their own accounts but all demos go together. Is that right?